book/src/versions/0.15/2023-04-27.md

```diff
@@ -1,26 +1,44 @@
-# 2023-03-28 Updates
+# 2023-04-27 Updates
 
 We've published minor versions of a number of crates. This release is composed
-of bugfixes, performance improvements, and documentation improvements.
+of bugfixes, new features, performance improvements, and documentation
+improvements.
 
 ## Bugfixes
 
+Discord's Clyde AI bot has a unique avatar hash. Unlike other hashes that are
+hex based, Clyde's avatar hash is simply "clyde". We now handle deserialization
+of Clyde's avatar in our [`ImageHash`] optimization.
+
+Application command interaction options of type `String` are no longer trimmed
+of leading zeroes.
+
+The [`UpdateCurrentMember`] request now correctly nulls the current member's
+nickname if passed `None`.
+
 The [gateway queue]'s [`DayLimiter`] now properly calculates when the session
 resets.
 
 The [`UpdateGuildSticker`] request now uses the specified audit log reason;
 prior, it was accidentally ignoring the reason.
 
-Discord's Clyde AI bot has a unique avatar hash. Unlike other hashes that are
-hex based, Clyde's avatar hash is simply "clyde". We now handle deserialization
-of Clyde's avatar in our [`ImageHash`] optimization.
-
```

Comment:
Not complaining, but is there a reason this was moved up?

Reply:
See my updated PR description